samples/matter/common/src/board/board.cpp

```diff
@@ -18,6 +18,8 @@
 
 LOG_MODULE_DECLARE(app, CONFIG_CHIP_APP_LOG_LEVEL);
 
+namespace Nordic::Matter {
```

Comment:
```suggestion
namespace Nordic {
```
I'd like to keep Matter namespace for more Matter related APIs, like init and event handling.

Reply:
Ok, aligned in all places.